ocean_provider/util_url.py

```diff
@@ -156,6 +156,8 @@ def check_url_details(url, with_checksum=False):
         pass
     except requests.exceptions.ConnectionError:
         pass
+    except requests.exceptions.ReadTimeout:
+        pass
 
```

Comment:
Can we have also a generic exception?
What if we will find another kind of exception in 2 days ?